CI_YAML.md

```diff
@@ -165,11 +167,14 @@ tags: >
               {"dependency": "open_jdk", "version": "11"}
           ]
         timeout: 60
-    ```
-    - Send PR, wait for the checks to go green (the change takes effect on presubmit)
+      ```
+    - Send PR, wait for the checks to go green (**the change takes effect on presubmit**)
 3. If the check is red, add patches to get it green
 4. Once the PR has landed, infrastructure may take 1 or 2 commits to apply the latest properties
```

Comment:
I'm confused; it takes effect immediately for presubmit, but not for postsubmit? So property changes still can't be landed in the same PR as changes that require them (like removing bring up)?

Or is this not the case for `properties`?

Reply:
So item 4 here should be removed, right?

Reply:
We still need it. The `properties` will be seen by the current PR/commit immediately, but there are cases where we need to wait, such incoming PRs/commits, which haven't rebased on top of that changing PR, and local LED runs.

Expanded a bit. Please let me know if this makes sense to you.

Reply:
Another case is caches. While dependencies can be upgraded, there may be some slow downs on the initial roll out due to cache invalidation

Reply:
Added one more item to cover.